en/docs/design/create-api/create-a-streaming-api-from-an-asyncapi-definition.md

```diff
@@ -0,0 +1,109 @@
+
+    <html><div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>The <b>CREATE API</b> button will only appear if the user who has signed in has the creator role permission.</p>
+      </div>
+    </html>
+
+2. Click **CREATE API** and then click **Import AsynAPI Definition**.
+
+    [![]({{base_path}}/assets/img/learn/design-api/streaming-api-from-asyncapi/import-asyncapi-definition.png)]({{base_path}}/assets/img/learn/design-api/streaming-api-from-asyncapi/import-asyncapi-definition.png)
+
+3. Select one of the following options:
+
+	* **AsyncAPI URL** - If you select this option, you need to provide a URL.
+	* **AsyncAPI File** - If you select this option, click **Browse File to Upload** and upload a file which contains an AsyncAPI definition.
+
+4.  Select **AsyncAPI URL**, provide the following URL, and click **Next**. 
+
+     ```sh
+     https://raw.githubusercontent.com/asyncapi/asyncapi/master/examples/2.0.0/streetlights.yml
+     ```
+
+     [![]({{base_path}}/assets/img/learn/design-api/streaming-api-from-asyncapi/websocket/websocket-streaming-api-from-asyncapi-url-import.png)]({{base_path}}/assets/img/learn/design-api/streaming-api-from-asyncapi/websocket/websocket-streaming-api-from-asyncapi-url-import.png)
+
+5.  Edit the information as given below and click **Create**.
+
+     | **Field**   | **Sample value** |
+     |-------------|------------------|
+     | Name | StreetlightsAPI |
+     | Context | /streetlights |
+     | Version | 1.0.0 |
+     | Protocol | WebSocket (or any other type of Streaming API) |
+     | Endpoint | ws://localhost:8080 |
+ 
+     [![]({{base_path}}/assets/img/learn/design-api/streaming-api-from-asyncapi/websocket/websocket-streaming-api-from-asyncapi-configure-values.png)]({{base_path}}/assets/img/learn/design-api/streaming-api-from-asyncapi/websocket/websocket-streaming-api-from-asyncapi-configure-values.png)
+
+     The **StreetlightsAPI API** overview page appears.
+
+     [![]({{base_path}}/assets/img/learn/design-api/streaming-api-from-asyncapi/websocket/websocket-streaming-api-from-asyncapi-overview.png)]({{base_path}}/assets/img/learn/design-api/streaming-api-from-asyncapi/websocket/websocket-streaming-api-from-asyncapi-overview.png)
+
+### Topics
+   
+Click **Topics** to navigate to the topics page.
+
+[![]({{base_path}}/assets/img/learn/design-api/streaming-api-from-asyncapi/websocket/websocket-streaming-api-from-asyncapi-topics.png)]({{base_path}}/assets/img/learn/design-api/streaming-api-from-asyncapi/websocket/websocket-streaming-api-from-asyncapi-topics.png)
+
+You will notice that the topics have been created automatically, from the AsyncAPI definition specified in the provided URL.
+
+
+## Subscriptions
+
+1. Click **Subscriptions**.
+
+2. Select **AsyncGold** and **AsyncSilver** as the Business plans. If you have chosen WebSub as the Streaming API type, choose **AsyncWHGold** and **AsyncWHSilver**.
```

Comment:
Shouldn't we have to mention about AsyncBronze and unlimited tiers. @sajithshn are we going with these names?

Reply:
Since this is like an example, we don't need to refer AsyncBronze and Unlimited. (For eg, similar thing has been done in REST APIs https://apim.docs.wso2.com/en/latest/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition/#subscriptions).